src/components/playback/playbackmanager.js

```diff
@@ -1848,6 +1848,31 @@ class PlaybackManager {
                     SortBy: options.shuffle ? 'Random' : 'SortName',
                     MediaTypes: 'Audio'
                 });
+            } else if (firstItem.Type === 'Series') {
+                promise = new Promise(function (resolve, reject){
+                    const apiClient = ServerConnections.getApiClient(firstItem.ServerId);
+                    apiClient.getEpisodes(firstItem.Id, {
+                    IsVirtualUnaired: false,
+                    IsMissing: false,
+                    UserId: apiClient.getCurrentUserId(),
+                    Fields: 'Chapters'
+                    }).then(function (episodesResult) {
+                        let foundItem = false;
+                        episodesResult.Items = episodesResult.Items.filter(function (e) {
+                            if (foundItem) {
+                                return true;
+                            }
+                            if (!e.UserData.Played) {
+                                foundItem = true;
+                                return true;
+                            }
+
+                            return false;
+                        });
+                        episodesResult.TotalRecordCount = episodesResult.Items.length;
+                        resolve(episodesResult);
+                    }, reject);
+                });
```

Comment:
```suggestion
                        return episodesResult;
                });
```

Reply:
Ah thanks, though when it comes to `'Season'` the interaction has to be different since one should queue all episodes after the present one. A separate future PR makes sense methinks.